### ISSUE TITLE
Entitlements hook file with problems

### ISSUE THREAD

**Issue by allanpoppe** [jcesarmobile/FilePicker-Phonegap-iOS-Plugin#48]
If entitlements file exists, the hook stops running and never defers.

>     var destFile = path.join(iosFolder, projName, 'Resources', projName + '.entitlements');
>     if (fs.existsSync(destFile)) {
>       console.error("File exists, not doing anything: " + destFile);
>     } else {

There are 2 problems here: 
1 - The entitlements file can exist and don't have FilePicker necessary entitlements, created by another plugin;
2 - I think the hook must defer even if nothing was to be done, otherwise `cordova platform add` stops running after this.

I don't know what has to be done, I'm not familiar with hook nor entitlements logics.

**Comment by jcesarmobile** [comment]
Yeah, I was waiting for Cordova to switch to the usage of .entitlement files and use edit-config tag to write on it and remove the hook, but I should probably write in the current entitlement.plist files and remove the hooks

**Comment by allanpoppe** [comment]
@jcesarmobile https://cordova.apache.org/docs/en/latest/plugin_ref/spec.html#edit-config

An example:
```
        <edit-config file="*-Info.plist" mode="merge" target="NSCameraUsageDescription">
            <string>Você pode fazer upload de imagens através deste app.</string>
        </edit-config>
```

Isn't this what you're looking for?

**Comment by allanpoppe** [comment]
Oh, I see. Sorry for my ignorance.

I think you should write in the entitlements.plist, indeed. 
This is giving me a hell of a headache, and I imagine others are struggling with this too. 

I'm having to take another plugin (branch-cordova-sdk) from the config.xml before each time I run `platftorm add`, and reinsert it after. This is because it also uses the same entitlements file and writes in it before FilePicker does.

Thank you for your effort on this, this was the only real solution for picking files on iOS I could find. It would be awesome if you did this update.
